hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockOutputStreamEntry.java

```diff
@@ -291,8 +297,9 @@ private BlockID underlyingBlockID() {
    *                   futures if false.
    * @return
    */
-  public boolean checkStreamFailures(boolean forPutBlock) {
+  public List<ECBlockOutputStream> getFailedStreams(boolean forPutBlock) {
     final Iterator<ECBlockOutputStream> iter = blockStreams().iterator();
+    List<ECBlockOutputStream> failedStream = new ArrayList<>();
```

Comment:
Nit: It would read better if `failedStream` was named `failedStreams` as it is a list of streams rather than a single stream.

Reply:
done. Thanks